src/assets/configuration/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "root": {
     "applicationLogo": "assets/ngen.png",
-    "applicationTitle": "NGenius Gaming",
+    "applicationTitle": "NGenius Gaming Team View",
```

Comment:
This little change broke the corresponding test 😝 

https://github.com/NGeniusGaming/ngenius-teamview/pull/8/checks?check_run_id=418296808#step:5:110

Reply:
Tests are fixed